spec.md

```diff
@@ -223,41 +228,55 @@ License: GPL-3.0-or-later
 
 ## Order of precedence
 
-Copyright and Licensing Information is considered according to the
-following order of precedence:
+If a Commentable File contains Copyright and Licensing Information but also has
+an adjacent `.license` file, then the Copyright and Licensing Information
+defined in the `.license` file takes precedence, and the Commentable File's
+contents are ignored.
 
-1. Information defined in the `.license` file.
-2. Information defined in the Commentable File.
-3. Information defined in `.reuse/dep5`.
+Copyright and Licensing Information defined in `.reuse/dep5` is aggregated with
+the Copyright and Licensing Information found in the covered files or in their
+adjacent `.license` files. For clarity, this means that if the SPDX License
+Expressions in a file's comment header and in the section for that file in
+`.reuse/dep5` do not agree with each other, then both SPDX License Expressions
+will apply to the file.
 
-There is no merging of information from different sources. Only the
-source with the highest precedence is considered.
+## Format of Copyright Notices
 
-## Format of copyright notices
+A Copyright Notice MUST start with a tag, word or symbol (collectively:
+prefixes) from the following list:
 
-A copyright notice MUST be prefixed by a tag, symbol or word denoting a
-copyright notice as described in this specification.
+- `SPDX-FileCopyrightText` (or `SPDX-SnippetCopyrightText` in Snippets)
+- Copyright
+- ©
 
-The copyright notice MUST contain the name of the copyright holder. The
+It is RECOMMENDED to use the `SPDX-FileCopyrightText` tag. You MAY add '(C)' or
+'©' after the prefix.
+
+A Copyright Notice MUST be terminated by a newline.
+
+The Copyright Notice MUST contain the name of the copyright holder. The
 copyright notice SHOULD contain the year of publication and the contact address
 of the copyright holder. The order of these items SHOULD be: year, name, contact
 address.
 
 The year of publication MAY be a single year, multiple years, or a span of
 years.
 
-The copyright holder MAY be an individual, list of individuals, group, legal
-entity, or any other descriptor by which one can easily identify the
-copyright holder(s).
+The copyright holder SHOULD be an individual, list of individuals, group, legal
+entity, or any other descriptor by which one can easily identify the copyright
+holder(s).
 
 Any contact address SHOULD be in between angle brackets.
 
-Examples of valid copyright notices:
+You MAY add any further information to the Copyright Notice.
+
+Examples of valid Copyright Notices:
 
 ```
 SPDX-FileCopyrightText: 2019 Jane Doe <jane@example.com>
 SPDX-FileCopyrightText: © 2019 John Doe <joe@example.com>
+SPDX-SnippetCopyrightText: (C) Example Cooperative <info@coop.example.com>
 © Example Corporation <https://corp.example.com>
 Copyright 2016, 2018-2019 Joe Anybody
```

Comment:
Can we finally get rid of the Copyright Notices that are without `SPDX-*CopyrightText`?

Reply:
Let's discuss this at the 26 October coordinator meeting.

I propose we delay that for the 4.0 spec, because it's a breaking change.

Reply:
Requiring that prefix makes adopting this standard harder for older projects, so if it happens, please don't do it at a point release.

Reply:
Absolutely agree that this should be a major release change.